OctoKit/Configuration.swift

```diff
@@ -13,7 +13,7 @@ public struct TokenConfiguration: Configuration {
     public var apiEndpoint: String
     public var accessToken: String?
     public let errorDomain = OctoKitErrorDomain
-    public let authorizationHeader: String? = "Basic"
+    public let authorizationHeader: Optional<String>
```

Comment:
This change wasn't discussed, but is necessary to allow the second initializer to set `authorizationHeader`.

Alternative would be:
```swift
public private(set) var authorizationHeader: String? = "Basic"
```

Reply:
I prefer the alternative 😊

Reply:
Updated 👍